### ISSUE TITLE
VS2015 Community Edition won't start Node.JS after first build of project

### ISSUE THREAD

**Issue by Illsteward** [Microsoft/nodejstools#204]
Using Visual Studio 2015 Community Edition Release Candidate with nodejstools 1.1, web essentials and git addons, I get an error when I attempt to launch my Node.JS project repeatedly.
For the first time, the project builds and loads fine. After that, it crashes with no visible error. Turning the Node.js Exceptions On in Exception Settings dialog yelded this:
**Error(ENOENT) occurred
Message: Error: ENOENT, no such file or directory 'C:\Users\$(Username)\Documents\Visual Studio 2015\WebSites\ExpressApp1\node_modules\express.js'**
in fs.js, line 797 in function fs.statSync.
Express is installed normaly via npm as local dependency, configured and has already run from this project once. This occurs within subsequent launches of any project, even new ones, Also, after sucesful run, web.config file is added by IIS. However, this tries to start the project at nonsensical port (53365) instead of the provided 3000. Re-building the project or closing/opening either project, solution or Visual studio did nothing, as well as reinstall of both Node.JS and nodejstools.

**Comment by mousetraps** [comment]
Does everything work fine if you run ```node app.js``` (or whatever your startup file is) in the command line?

**Comment by Illsteward** [comment]
Yes, I can run the project multiple times from command line using node server just fine. I have tried several other projects and all work fine. Only problem that I have noticed outside of Visual Studio is that I have to start my projects with specified port because the proces.env.port which was defined when starting the project from Web Matrix now defaults to "Undefined". However the problem persists even after setting the port number manualy.
